resources/roca_lib.brs

```diff
@@ -68,6 +68,7 @@ function __roca_createDescribeBlock(mode as string, description as string, func
         describe: __roca_describe,
         fdescribe: __roca_fdescribe,
         xdescribe: __roca_xdescribe,
+        addFields: __roca_addFields
```

Comment:
We should probably add this to the [`roca` definition above](https://github.com/hulu/roca/blob/856874877d16cc56fffc0f6e9190393fb3894469/resources/roca_lib.brs#L7-L18) as well.

Reply:
I omitted adding it here because there isn't a suite object yet - so it works only within a `describe` sub.

Reply:
Figured it out